code/game/gamemodes/changeling/powers/regenerate.dm

```diff
@@ -28,7 +28,9 @@
 			C.regenerate_limbs(1)
 		C.regenerate_organs()
 		if(!user.getorganslot(ORGAN_SLOT_BRAIN))
-			var/obj/item/organ/brain/changeling_brain/B = new()
+			var/obj/item/organ/brain/B = new()
```

Comment:
Should this be the species’s brain typepath? In case of changeling nightmare/xeno?

Reply:
Yeah it should